webapp/i18n/en.json

```diff
@@ -2017,9 +2017,9 @@
   "team_members_dropdown.confirmDemotionCmd": "platform roles system_admin {username}",
   "team_members_dropdown.inactive": "Inactive",
   "team_members_dropdown.leave_team": "Remove From Team",
-  "team_members_dropdown.makeActive": "Make Active",
+  "team_members_dropdown.makeActive": "Activate",
```

Comment:
do any of these new strings need to be changed in the corresponding .jsx or .go files?

Otherwise looks good